hpc_launcher/systems/lc/el_capitan_family.py

```diff
@@ -88,13 +88,26 @@ def environment_variables(self) -> list[tuple[str, str]]:
                 )
             )
         if os.getenv("ROCM_PATH") is not None:
+            rocm_path = os.getenv("ROCM_PATH")
             env_list.append(
                 (
                     "LD_LIBRARY_PATH",
-                    os.path.join(os.getenv("ROCM_PATH"), "llvm", "lib")
+                    os.path.join(f"{rocm_path}", "llvm", "lib")
                     + ":${LD_LIBRARY_PATH}",
                 )
             )
+            rocm_ver = os.path.basename(rocm_path)
+            # Check for and include the AWS_OFI_PLUGIN if it exists
+            sys_type = os.getenv("SYS_TYPE")
+            aws_ofi_plugin = f'/collab/usr/global/tools/rccl/{sys_type}/{rocm_ver}/install/lib'
```

Comment:
Should this be enabled only when the backend is set to ccl?
Otherwise looks great!